ci/checks/style.sh

```diff
@@ -55,13 +55,12 @@ HEADER_META_RETVAL=$?
 echo -e "$HEADER_META"
 
 # Run doxygen and check for missing documentation
-DOXYGEN=`cd cpp/doxygen && { cat Doxyfile ; echo QUIET = YES; echo GENERATE_HTML = NO; }  | doxygen - 2>&1 | tail -n +3`
+DOXYGEN=`ci/checks/doxygen.sh`
```

Comment:
pre-commit will not run doxygen if there are no file changes in `cpp/include` in that PR. It may not run always.

Reply:
The CI command above is invoked with `--all-files` for this reason. It will always run, as we intend.

Try `pre-commit run --hook-stage manual --all-files` locally and you can see for yourself.

Reply:
removed doxygen check in style.sh, updated pre-commit to run in manual mode too.